Gemfile

```diff
@@ -11,7 +11,7 @@ gem "sprockets-rails"
 
 # Use sqlite3 as the database for Active Record
 gem "sqlite3", "~> 1.4"
-
+gem 'rails-controller-testing'
```

Comment:
Always lock your gems version so that you don't get conflicts in different environments